src/generators/html_generator.py

```diff
@@ -1,7 +1,36 @@
 import tmt
+from celery.result import AsyncResult
 from tmt import Test, Logger, Plan
 
 
+def generate_status_callback(r: AsyncResult, status_callback_url: str) -> str:
+    """
+    This function generates the status callback for the HTML file
+    :param r: AsyncResult object
+    :param status_callback_url: URL for the status callback
+    :return:
+    """
+    if r.status == "PENDING":
+        return (f'''<html>
+        <head>
+        <title>HTML File</title>
+        <meta charset="UTF-8">
+        </head>
+        <body>
+        Processing... Try this clicking this url in a few seconds: <a href="{status_callback_url}">{status_callback_url}
+        </a> </body>''')
+    else:
+        return (f'''<html>
+        <head>
+        <title>HTML File</title>
+        <meta charset="UTF-8">
+        </head>
+        <body>
+        Status: {r.status}<br>
+        The result is: <br>{r.result}
+        </body>''')
```

Comment:
Just a note: there is some html redundancy here that could be avoided using jinja templates. If you want this merged now and fixed later, it's fine for me. It's your preference @martinhoyer

Reply:
Not up to me, but make sense to me to use jinja where applicable. I would not use it in places, where save just a couple of lines though (imho).

Reply:
ok, after looking at it there are definitely some bits that could benefit from jinja.

Reply:
Added some jinja templates. Could you check it if you'll have a minute please? @inknos @seberm